vj4/model/builtin.py

```diff
@@ -67,6 +67,15 @@
 PERM_EDIT_CONTEST = 1 << 50
 PERM_EDIT_CONTEST_SELF = 1 << 51
 
+# Homework.
+PERM_VIEW_HOMEWORK = 1 << 52
```

Comment:
Note: `1 << 52` exceeds javascript int52 limit and `1 << 58` is near int64 limit.

Reply:
Seems it won't be exported to front-end.